README.md

```diff
@@ -48,8 +48,17 @@ You should be able to drag blocks from the list on the left into the center comp
 
 Getting up and running is really only the first step. You'll probably want to configure Mosaico to actually send email in a production environment. See the remaining sections for guidance.
 
+### Upgrading from version 1 to version 2
+
+We've introduced a new default image backend after 2.0.0. That means, if you are upgrading from v1, you will have to add the follow code in order to keep your current image backend. So that your won't have images missing. Create an initializer in config/initializers/mosaico.rb and add these lines:
```

Comment:
Spelling/grammar fixes:

> We've introduced a new default image backend since v2.0.0. If you are upgrading from v1, you will have to add the following code in order to keep your current image backend, i.e. to avoid broken images. Create an initializer in config/initializers/mosaico.rb and add these lines: